src/Spice86/Views/DisassemblyView.axaml

```diff
@@ -1,252 +1,235 @@
+                <Button Content="Go" Command="{Binding GoToAddressCommand}" CommandParameter="{Binding CurrentInstructionAddress}" />
+            </StackPanel>
+
+            <!-- Function Selection -->
+            <StackPanel Grid.Column="2" Orientation="Horizontal" Spacing="4" Margin="8,0,0,0" IsVisible="{Binding IsFunctionInformationProvided}">
+                <TextBlock VerticalAlignment="Center" Text="Function:" />
+                <AutoCompleteBox
+                    ItemsSource="{Binding Functions}"
+                    SelectedItem="{Binding SelectedFunction}"
+                    ItemFilter="{Binding FunctionFilter}"
+                    Watermark="Type to select function"
+                    ItemSelector="{Binding FunctionItemSelector}"
+                    GotFocus="OnFunctionSelectionFocus"
+                    KeyDown="OnFunctionSelectionKeyDown"
+                    MinWidth="200">
+                    <AutoCompleteBox.ItemTemplate>
+                        <DataTemplate DataType="debugging:FunctionInfo">
+                            <StackPanel Orientation="Horizontal">
+                                <TextBlock>
+                                    <Run Text="{Binding Name}" />
+                                    <Run Text=" [" /><Run Text="{Binding Address}" /><Run Text="]" />
+                                </TextBlock>
+                            </StackPanel>
+                        </DataTemplate>
+                    </AutoCompleteBox.ItemTemplate>
+                </AutoCompleteBox>
+                <Button x:Name="GoToFunctionButton"
+                        Command="{Binding GoToFunctionCommand}"
+                        CommandParameter="{Binding SelectedFunction}"
+                        IsEnabled="{Binding SelectedFunction, Converter={x:Static ObjectConverters.IsNotNull}}"
+                        Content="Go" />
+            </StackPanel>
+
+
+            <!-- View management-->
+            <StackPanel Grid.Column="4" Orientation="Horizontal" Spacing="4">
+                <Button
+                    HotKey="Ctrl+F4"
+                    ToolTip.Tip="Close view (Ctrl-F4)"
+                    Command="{Binding CloseTabCommand}"
+                    IsEnabled="{Binding CanCloseTab}">
+                    <fluent:SymbolIcon Symbol="DocumentDismiss" Width="20" Height="20" />
+                </Button>
+                <Button
+                    HotKey="Ctrl+N"
+                    ToolTip.Tip="New view (Ctrl-n)"
+                    Command="{Binding NewDisassemblyViewCommand}">
+                    <fluent:SymbolIcon Symbol="DocumentAdd" Width="20" Height="20" />
+                </Button>
+            </StackPanel>
+        </Grid>
+
+        <!-- Main content area with disassembly and registers -->
+        <Grid Grid.Row="1" ColumnDefinitions="*, Auto">
+            <!-- Loading indicator -->
+            <progRing:ProgressRing Grid.Column="0"
+                                   Width="40"
```

Comment:
Use MinWidth and MinHeight to allow for UI responsive design.

Reply:
Done.